Makefile

```diff
@@ -24,6 +24,9 @@ dependencies:
 	# We use 'go get' here because 'go install' will not build out of our vendored mock repo.
 	go get github.com/golang/mock/mockgen
 
+	# Necessary to generate thrift files, in addition to manually installing thrift via homebrew
+	go get github.com/apache/thrift/lib/go/thrift
```

Comment:
I'll remove it for now,  We may not need it.  I thought I did at one point trying to get it to work, but I think it may have just been my thrift version needed to be 0.9.3